vfs/vfs.go

```diff
@@ -231,6 +231,11 @@ func New(f fs.Fs, opt *vfscommon.Options) *VFS {
 	// with the same remote string we get this one. The Pin is
 	// removed when the vfs is finalized
 	cache.PinUntilFinalized(f, vfs)
+	
+	if vfs.Opt.Refresh {
+		fs.Debugf(f, "Refreshing VFS directory cache")
+		vfs.root.readDirTree()
```

Comment:
This is still synchronous. Run it in a goroutine by adding `go` in front of the statement to make it async.